content/post/data-dies-in-darkness.md

```diff
@@ -0,0 +1,73 @@
+---
+author: Michael Kaminsky
+title: "Data Dies in Darkness"
+date: "2018-02-25"
+tags: 
+ - analytics
+ - data quality
+ - org
+---
+
+# Data Dies in Darkness 
+### Building the Data Quality Flywheel
+
+
+The fastest way to doom an analytics team (and any hope of building a data-driven organization) is to present data and analyses that are often flawed or inconsistent. When people don't believe they can trust the data, they'll stop using them (and, if you're an analytics leader, you might be soon looking for a new job).
+
+There are two techniques I rely on for ensuring data and analytical quality:
+1. Analysis audit and code review
+2. The internal data quality flywheel
+
+While the first technique (good analysis review) is very important, in this post I'm going to focus on the second which is an incredibly powerful tool for thinking about not only adding value as a data and analytics team, but will pay ongoing dividends in data quality for years to come.
+
+When people talk about data quality, they often describe the *symptoms* of data quality issues. Like:
+* Reporting numbers changing over time
+* Unclear definitions of metrics
+* Inconsistent numbers across teams or reports
+
+These symptoms are generally driven by a few root issues:
+* Lack of a centrol repository of conformed data (single source of truth)
+* Disagreement or misalignment about how business phenomena or KPIs ought to be measured
+* Unnoticed "data drift" as business-logic assumptions or data integrations get stale
+
+My personal motto for improving data quality, adapted from [The Washington Post](https://www.washingtonpost.com/lifestyle/style/the-washington-posts-new-slogan-turns-out-to-be-an-old-saying/2017/02/23/cb199cda-fa02-11e6-be05-1a3817ac21a5_story.html?utm_term=.3b9fcbc03a44), is that "Data Dies in Darkness" (which is an appealingly more dramatic phrasing of "sunlight is the best disinfectant"). The easiest way to promote data quality is to have as many eyes on as many pieces of data as possible -- this way potential issues are quickly identified and fixed rather than "festering" and slowly degrading trust in the entire system.
```

Comment:
it feels like we should highlight this block as a pull quote or something.  it would be nice for anyone not familiar to get the context of the title at a glance.

Reply:
@swiftsam  How would you suggest doing this? (Feel free to commit directly to the branch or open a PR against this one)